app/controllers/v1/workflow/workflows_controller.rb

```diff
@@ -8,7 +8,8 @@ def show
   end
 
   def resources
-    @workflow = Workflow::Instance::Workflow.find_by!(external_identifier: params[:workflow_id])
+    query = params[:workflow_id] ? { external_identifier: params[:workflow_id] } : { id: workflow_id }
```

Comment:
This endpoint is `/v1/workflow/workflows/:workflow_id/resources`. Same question above if we should keep the optionality or not.

Reply:
keep for now